### DISCUSSION TITLE
我有一个想法

### DISCUSSION THREAD

**Discussion by daxiangzi166** [Spaso1/FindMaimaiDX_Phone#5] [Ideas]
亲爱的大佬您好
  我有一个不成熟的想法，就是能不能把查询添加一个可以查找中二节奏机台地址，还有能否再添加对于落雪查分器的支持
  谢谢大佬

**Comment by Spaso1** [comment]
好想法,未来应该会做,不过现在重心在可视化地图上面

**Reply by Spaso1** [reply]
现在最新开发版本已经实现可视化地图和手动设置准确信息,敬请期待喵

**Reply by daxiangzi166** [reply]
了解了
谢谢大佬

**Reply by Spaso1** [reply]
已经完成落雪查分器查分支持，上传估计要等落雪那边更新新功能![IMG_20241204_181222.jpg](https://github.com/user-attachments/assets/4ae15007-5516-4aba-86ae-5a8b3661aeff)
